packages/office-addin-debugging/src/start.ts

```diff
@@ -385,6 +385,18 @@ export async function startDebugging(options: StartDebuggingOptions) {
     }
 }
 
+/**
+ * @param option 
+ * @param other 
+ * @returns other if option is undefined or option
+ */
+function definedOrOther<T>(option:T | undefined, other: T) {
```

Comment:
This function is not really needed. You could just as easily write:

```
debuggingMethod: options.DebuggingMethod || defaultDebuggingMethod(),
enableDebugging: options.EnableDebugging || true,
```

Reply:
/Azurepipelines run

Reply:
@akrantz 

In this case perhaps, but it's easy to duplicate and to then encounter an edge case that does not perform as expected:

```typescript
const va = 0 || "hello"
console.log(va) // hello

const vb = false || "hello"
console.log(vb) // hello
```

Reply:
well really the point is to check whether undefined or not, rather than whether falsey or not. Personally the function didn't make that clear. Perhaps it was the name. 

Here's the best way in TypeScript:

`const value = options.DebuggingMethod ?? true;`